### PR TITLE
Fix stops layer rendering

### PR DESCRIPTION
**Pull request by philip-cline**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JSDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [x] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

Previously because of how we were rendering Location Groups, we began rendering all stops when the stops menu was opened. This created major performance issues with Leaflet. This PR stops this process, but maintains the previous location groups behaviour.

### CODE REVIEW

**Comment by philip-cline** [comment]
e2e tests got scary there for a bit but think that should be resolved now

**Review by daniel-heppner-ibigroup — APPROVED** [review]
Thanks for the emoji

File: __tests__/e2e/docker-compose.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ services:
       args: *common-variables
       dockerfile: ./__tests__/e2e/Dockerfile
 
-    command: /bin/sh -c "./wait-for-command.sh -t 15 -c 'nc -z datatools-ui 9966' && sleep 25 && yarn test-end-to-end"
+    command: /bin/sh -c "./wait-for-command.sh -t 15 -c 'nc -z datatools-ui 9966' && sleep 40 && yarn test-end-to-end"
```
Comment (miles-grant-ibigroup) [inline __tests__/e2e/docker-compose.yml]: I've changed my mind and I actually think we should revert this change after all. e2e tests seem to be fickle but not because of this.

**Review by miles-grant-ibigroup — APPROVED** [review]
Great fix thanks for cleaning that up

File: __tests__/e2e/docker-compose.yml (hunk 1 above)
Comment (philip-cline) [reply __tests__/e2e/docker-compose.yml]: Yeah agreed, I'll revert rn
